lib/wick-strict-run

```diff
@@ -30,9 +30,16 @@ wickStrictRun() {
     shift
     setOptions=$-
     set +eE
+    # Turning off the error trap for this context.  I don't
```

Comment:
Newline above.

Some of this comment should be in the doc block so it makes it into the generated documentation.